src/components/atoms/OffcanvasHeader/OffcanvasHeader.js

```diff
@@ -47,13 +47,13 @@ export default class OffcanvasHeader extends HTMLElement {
     const offcanvasHeaderClasses = ['offcanvas-header'];
     this.offcanvasTitle.className = 'offcanvas-title';
     this.offcanvasTitle.id = `${parentID}-label`;
-    this.closeBtn.setAttribute('data-img-alt', '');
-    this.closeBtn.setAttribute('data-icon', '');
-    this.closeBtn.setAttribute('data-close', 'true');
-    this.closeBtn.setAttribute('data-bs-dismiss', parentID);
+    this.closeBtn.setAttribute('img-alt', '');
+    this.closeBtn.setAttribute('icon', '');
+    this.closeBtn.setAttribute('close', 'true');
+    this.closeBtn.setAttribute('bs-dismiss', parentID);
```

Comment:
In ModalFooter we left this attribute unchanged but in ModalHeader and here we removed `data-` prefix. Which is correct?